docs/source/user-guide/sql/aggregate_functions.md

```diff
@@ -281,11 +281,11 @@ first_value(expression [ORDER BY expression])
 #### Example
 
 ```sql
-> SELECT first_value(column_name ORDER BY other_column) FROM table_name;
+> SELECT last_value(column_name ORDER BY other_column) FROM table_name;
```

Comment:
the documentation fixed now. 
There is an existing bug when documentation lock with the same name covers 2 or more doc functions in the same file. In this scenario documentation will pick the first available doc description which is unexpected. The migration to macros fixes the problem as it uses separate lock per function